besu/src/main/java/org/hyperledger/besu/cli/options/stable/DataStorageOptions.java

```diff
@@ -122,6 +122,14 @@ public static class Unstable {
             "Enables code storage using code hash instead of by account hash. (default: ${DEFAULT-VALUE})")
     private boolean bonsaiCodeUsingCodeHashEnabled = DEFAULT_BONSAI_CODE_USING_CODE_HASH_ENABLED;
 
+    @CommandLine.Option(
+        hidden = true,
+        names = {"--Xbonsai-parallel-tx-enabled"},
+        arity = "1",
+        description =
+            "Enables parallelization of transactions to optimize processing speed by concurrently loading and executing necessary data in advance. (default: ${DEFAULT-VALUE})")
+    private Boolean isParallelTxEnabled = false;
```

Comment:
non blocking suggestion: s/ParallelTxEnabled/ParallelTxProcessingEnabled/
seems more appropriate to specify to what the parallelism apply

Reply:
done.